python/tests/voice/test_elevenlabs_pump.py

```
@@ -0,0 +1,469 @@
+    from scenario.voice.adapter import VoiceAgentAdapter
+
+    class _Base(VoiceAgentAdapter):
+        async def connect(self) -> None: ...
+        async def disconnect(self) -> None: ...
+        async def send_audio(self, chunk: AudioChunk) -> None: ...
+        async def recv_audio(self, timeout: float) -> AudioChunk:
+            return AudioChunk(data=b"")
+
+    assert _Base().is_connected() is True
+
+    adapter, socket = await _connected_adapter()
```

Ruff (0.15.20)

[warning] Unpacked variable `socket` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)